modular_skyrat/modules/nurse/code/nurse.dm

```diff
@@ -0,0 +1,60 @@
+/datum/job/nurse
+	title = "Nurse"
+	department_head = list("Chief Medical Officer")
```

Comment:
you need spawn points for these mofos, and by that I mean put this on the job:
``job_spawn_title = "Medical Doctor"``

Reply:
A'ight.